settings.py

```diff
@@ -4,11 +4,22 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-store: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-MONGODB_URL = os.getenv("MONGODB_URL")
+DB_HOST = os.getenv('POSTGRES_HOST', '0.0.0.0')
```

Comment:
Due to the way the config works, I think we should name these env variable SDX_STORE_POSTGRES_HOST to avoid naming clashes